### PR TITLE
modified tooltip content for username and name attribute on identity management step

### PR DESCRIPTION
**Pull request by raymondz1**
### What this PR does / why we need it
The search logic on identity management step is hard coded. No matter what we put for Username for the User Search Attributes or Name Attribute for the Group Search Attributes, when we try to verify the ldap configuration, it ends up searching by CN for User Search (https://github.com/vmware-tanzu/tanzu-framework/blob/main/pkg/v1/tkg/ldap/client.go#L111) and OU for Group Search (https://github.com/vmware-tanzu/tanzu-framework/blob/main/pkg/v1/tkg/ldap/client.go#L180). The existing tooltip can't find any information regarding this, so the tooltip is modified to provide more details in the PR.
### Which issue(s) this PR fixes

Fixes https://github.com/vmware-tanzu/tanzu-framework/issues/1269

### Release note

```release-note

```

### PR Checklist

- [ ] Squash the commits into one or a small number of logical commits
      
- [ ] Use good commit [messages](https://github.com/vmware-tanzu/tanzu-framework/blob/main/CONTRIBUTING.md)
- [ ] Ensure PR contains terms all contributors can understand and links all contributors can access

### CODE REVIEW

File: pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html [hunk 1]
```diff
@@ -403,7 +403,7 @@ <h5 i18n="ldap user search attributes title" aria-level="3">
                             <clr-tooltip>
                                 <clr-icon aria-live="assertive" a11yTooltipTrigger shape="info-circle" size="22"></clr-icon>
                                 <clr-tooltip-content clrPosition="top-left" clrSize="lg" *clrIfOpen>
-                                    <span>The LDAP attribute that contains the username used in Kubernetes. In AD environments
+                                    <span>The LDAP attribute that contains the username used in Kubernetes and searched by CN only for now. In AD environments
```
Comment (miclettej) [inline pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html:406]: Please change to:

The LDAP attribute that contains the username used in Kubernetes; can only be searched by CN.

File: pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html [hunk 2]
```diff
@@ -477,12 +477,12 @@ <h5 i18n="group user search attributes title" aria-level="3">
                             <clr-tooltip>
                                 <clr-icon aria-live="assertive" a11yTooltipTrigger shape="info-circle" size="22"></clr-icon>
                                 <clr-tooltip-content clrPosition="top-left" clrSize="lg" *clrIfOpen>
-                                    <span>The LDAP attribute that Kubernetes will see as part of your groups (e.g., cn).</span>
+                                    <span>The LDAP attribute that Kubernetes will see as part of your groups (e.g., ou). It is searched by OU only for now</span>
```
Comment (miclettej) [inline pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html:480]: Please change to:

The LDAP attribute that Kubernetes will see as part of your groups (e.g., ou); can only be searched by OU.

**Review by miclettej — APPROVED** [review]

File: pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html (hunk 1 above)
Comment (raymondz1) [reply pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html]: fixed

File: pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html (hunk 2 above)
Comment (raymondz1) [reply pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html]: fixed

**Pull request merged**